src/detections/detection.rs

```diff
@@ -41,29 +46,59 @@ impl Detection {
+                                            &event.user_data,
+                                            event_data,
+                                        );
+                                    }
+                                    _ => println!("Not Match is Event ID Num:{}", event_id),
+                                }
                             } else if channel == "System" {
-                                &system.detection(event_id, &event.system, event_data);
+                                match event_id.as_str() {
+                                    "4688" | "4672" | "4720" | "4728" | "4732" | "4756"
+                                    | "4625" | "4673" | "4674" | "4648" | "1102" => {
+                                        &system.detection(event_id, &event.system, event_data);
+                                    }
+                                    _ => println!("Not Match is Event ID Num:{}", event_id),
+                                }
                             } else if channel == "Application" {
-                                &application.detection(event_id, &event.system, event_data);
+                                match event_id.as_str() {
+                                    "2" => {
+                                        &application.detection(event_id, &event.system, event_data);
+                                    }
+                                    _ => println!("Not Match is Event ID Num:{}", event_id),
+                                }
                             } else if channel == "Microsoft-Windows-PowerShell/Operational" {
-                                &powershell.detection(event_id, &event.system, event_data);
+                                match event_id.as_str() {
+                                    "8003" | "8004" | "8006" | "8007" => {
+                                        &powershell.detection(event_id, &event.system, event_data);
+                                    }
+                                    _ => println!("Not Match is Event ID Num:{}", event_id),
+                                }
                             } else if channel == "Microsoft-Windows-Sysmon/Operational" {
-                                &sysmon.detection(event_id, &event.system, event_data);
+                                match event_id.as_str() {
+                                    "4103" | "4104" => {
+                                        &sysmon.detection(event_id, &event.system, event_data);
+                                    }
+                                    _ => println!("Not Match is Event ID Num:{}", event_id),
+                                }
                             } else if channel == "Microsoft-Windows-AppLocker/EXE and DLL" {
-                                &applocker.detection(event_id, &event.system, event_data);
+                                match event_id.as_str() {
+                                    "1" | "7" => {
```

Comment:
channelはL95でAppLockerとなっていますが、マッチさせているeventIDがSysmonのものになっています。
以下のように変更していただくと正しいと思います。

```suggestion
                                    "8003" | "8004" | "8006" | "8007" => {
```

Reply:
46022cc により修正をしました。